packages/strapi-plugin-users-permissions/services/UsersPermissions.js

```diff
@@ -559,7 +559,7 @@ module.exports = {
     );
 
     // stringify mongoDB _id for add/remove matching
-    if (role._id ? '_id' : 'id' === '_id') {
+    if (role._id) {
```

Comment:
thought the plan was to remove references to `_id` and just use `id` directly

Reply:
that's not the goal of that PR. Just cleaning this code because it's hard to understand

Reply:
that's not the goal of that PR. Just cleaning this code because it's hard to understand